packages/amplify-category-auth/src/provider-utils/supported-services.ts

```diff
@@ -1240,7 +1240,7 @@ const supportedServices = {
       },
       {
         key: 'signinwithapplePrivateKeyUserPool',
-        question: 'Enter your Private Key for your OAuth flow: ',
+        question: 'Enter your Apple Private Key for your OAuth flow: ',
```

Comment:
and here
![image](https://user-images.githubusercontent.com/110861985/198719306-e3d13fd9-6eee-4cfa-959b-c9c4b99777ba.png)